jaeger.sh

```diff
@@ -0,0 +1,6 @@
+docker run \
```

Comment:
maybe add a comment what's the purpose of this script on the first line? 
like `# run jaeger for viewing traces when debugging locally` (or something like that?)